### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by mauriceackel**
**What changes does this PR introduce**

Adds more translations, makes Google Maps key an environment variable

### CODE REVIEW

**Review by kenodressel — APPROVED** [review]
legit

File: MAINTAINERS.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ QuarantineHeros is maintained by the following group of people:
 - Mail [keno.dressel@quarantaenehelden.org](mailto:keno.dressel@quarantaenehelden.org) 
 
 **Maurice**
-- GitHub [@Maurice22](https://github.com/Maurice22)
+- GitHub [@mauriceackel](https://github.com/mauriceackel)
```
Comment (jagaSto) [inline MAINTAINERS.md:17]: damn son, did you change your GitHub name? are you a madman? @mauriceackel :)

File: firebase/functions/src/domain/onUserDelete.ts [hunk 2]
```diff
@@ -45,16 +45,18 @@ export async function onUserDelete(user: UserRecord): Promise<void> {
...
-    reportedEntries.forEach((doc) => {
-      if (!doc.exists) return;
-
-      promises.push(doc.ref.update({ 'd.reportedBy': admin.firestore.FieldValue.arrayRemove(user.uid) }));
-      const data = doc && doc.data();
-      if (data && !data.d.reportedBy.includes('ghost-user')) {
-        promises.push(doc.ref.update({ 'd.reportedBy': admin.firestore.FieldValue.arrayUnion('ghost-user') }));
-      }
-    });
+    if(entryRefs.length > 0) {
```
Comment (jagaSto) [inline firebase/functions/src/domain/onUserDelete.ts:48]: can `entryRefs` be `undefined` here?

**Review by jagaSto — APPROVED** [review]
1 small question :)

File: firebase/functions/src/domain/onUserDelete.ts (hunk 2 above)
Comment (mauriceackel) [reply firebase/functions/src/domain/onUserDelete.ts:48]: No, it will at least be the empty array.

**Pull request merged**